wizeline/remix-project-lab
# feat: change tab icon

# What does this PR do?
Change the tab icon and fixed the skill filter error from the cards


# Where should the reviewer start?
app/core/components/ProposalCard/index.tsx



# How should this be manually tested?
To test the tab icon, you should see it at any page 
To test the filter issue,..
1.- Click in a skill of any card 



# Any background context you want to provide?
Change the tab icon


# What are the relevant tickets?
https://github.com/orgs/wizeline/projects/41/views/1?pane=issue&itemId=26713549


# Screenshots
Before
<img width="116" alt="Screenshot 2023-10-26 at 9 47 57" src="https://github.com/wizeline/remix-project-lab/assets/97254325/c29c51a2-9e57-4bac-8f69-db13e1ecac02">

After
<img width="81" alt="Screenshot 2023-10-26 at 9 55 08" src="https://github.com/wizeline/remix-project-lab/assets/97254325/079567fd-8a0b-4c75-b983-37d911b8e534">

# Questions



# Checklist
